sources/packages/web/src/views/institution/student/applicationDetails/InstitutionAssessmentsSummary.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <full-page-container layout-template="centered">
+    <template #header>
+      <header-navigator
+        title="Student applications"
+        :routeLocation="{
+          name: InstitutionRoutesConst.STUDENT_APPLICATIONS,
```

Comment:
The whole route can be returned to template as variable like in StudentAppealRequestsApproval.vue